src/index.rs

```diff
@@ -94,6 +95,44 @@ impl IndexWriter {
     fn commit_opstamp(&self) -> u64 {
         self.inner_index_writer.commit_opstamp()
     }
+
+    /// Delete all documents containing a given term.
+    ///
+    /// Args:
+    ///     field_name (str): The field name for which we want to filter deleted docs.
+    ///     field_value (PyAny): Python object with the value we want to filter.
+    ///
+    /// If the field_name is not on the schema raises ValueError exception.
+    /// If the field_value is not supported raises Exception.
+    fn delete_documents(
+        &mut self,
+        field_name: &str,
+        field_value: &PyAny,
+    ) -> PyResult<u64> {
+        if let Some(field) = self.schema.get_field(field_name) {
```

Comment:
I tend to like `.ok_or_else` for that kind of stuff nowadays (I know my code is inconsistent on that point).
It removes one level and indentation and matches the way people would explain what happens.

```rust
let field = self.schema.get_field(field_name)
               .ok_or_else(|| exceptions::ValueError::py_err(format!(...)))?;
...

Reply:
This is largely a matter of taste... so feel free to not do anything and makr the comment as resolved.